bindings/gumjs/gumcmodule.c

```diff
@@ -396,51 +478,50 @@ gum_cmodule_new (const gchar * source,
 void
 gum_cmodule_free (GumCModule * cmodule)
```

Comment:
We should delete this and use `g_object_unref ()` instead. (And `g_clear_object (&p)` when needing to discard in nullable cases.)

Reply:
I used `g_clear_object` where possible - this might be unnecessary in some locations, but I think the overhead is negligible.